src/test/java/com/znsio/rpap/steps/AppSteps.java

```diff
@@ -0,0 +1,56 @@
+package com.znsio.rpap.steps;
+
+import com.applitools.eyes.appium.Eyes;
+import com.applitools.eyes.appium.Target;
+import com.epam.reportportal.annotations.Step;
+import com.znsio.rpap.pages.AppExample;
+import io.appium.java_client.AppiumDriver;
+import org.apache.log4j.Logger;
+import org.testng.Assert;
+
+import static com.znsio.reportportal.integration.utils.ReportPortalLogger.captureAndAttachScreenshot;
+import static com.znsio.reportportal.integration.utils.ReportPortalLogger.logInfoMessage;
+
+public class AppSteps {
+
+    private static final Logger LOGGER = Logger.getLogger(AppSteps.class.getName());
+    private AppiumDriver appDriver;
+    private AppExample appPage;
+    private Eyes eyesOnApp;
+
+    public AppSteps(AppiumDriver appDriver, AppExample appPage, Eyes eyesOnApp) {
+        this.appDriver = appDriver;
+        this.appPage = appPage;
+        this.eyesOnApp = eyesOnApp;
+    }
+
+    @Step
+    public void performOperation(String input1, String operator1, String input2, String operator2) {
+        appPage.chooseNumber(input1);
+        log("Selected No." + input1);
+        appPage.chooseOperation(operator1);
+        log("Select Calculator Operation " + operator1);
+        appPage.chooseNumber(input2);
+        log("Select No." + input2);
+        captureAndAttachScreenshot(appDriver, "Data entered on Calculator");
+        eyesOnApp.check(Thread.currentThread().getStackTrace()[1].getMethodName(), Target.window());
+        log("Select Calculator Operation " + operator2);
+        appPage.chooseOperation(operator2);
+    }
+
+    @Step
+    public void verifyResult(String expected) {
+        log("Expected output " + expected);
+        String actualOutput = appPage.getCalculatorDisplay();
+        log("Actual output " + actualOutput);
+        Assert.assertEquals(expected, actualOutput);
+        captureAndAttachScreenshot(appDriver, "Final result captured");
+        eyesOnApp.check(Thread.currentThread().getStackTrace()[1].getMethodName(), Target.window());
```

Comment:
I've noticed on the Applitools dashboard that the tests are failing intermittently because of one extra element which sometime is visible on the calculator result screen. Why are we not handlinf it?

Reply:
Need to discuss

Reply:
After discussion we deicided to keep dialpad screenshot of operand and initial calculator screen.
Please refer this screenshot

https://jioeyes.applitools.com/app/test-results/00000251705726447379/?accountId=8euMeI_LvUiaNtUWS_HwsA__